site/src/pages/CreateWorkspacePage/ExternalAuthBanner/ExternalAuthBanner.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import { TemplateVersionExternalAuth } from "api/typesGenerated";
+import { ExternalAuthBanner } from "./ExternalAuthBanner";
+import type { Meta, StoryObj } from "@storybook/react";
+
+const MockExternalAuth: TemplateVersionExternalAuth = {
+  id: "",
+  type: "",
+  display_name: "GitHub",
+  display_icon: "/icon/github.svg",
+  authenticate_url: "",
+  authenticated: false,
+};
+
+const meta: Meta<typeof ExternalAuthBanner> = {
+  title: "pages/CreateWorkspacePage/ExternalAuthBanner/ExternalAuthBanner",
```

Comment:
I don't think adding the extra folder to storybook adds any benefit personally, and would suggest just naming it `"pages/CreateWorkspacePage/ExternalAuthBanner"`